Bindings/Java/tests/TestEditProperties.java

```diff
@@ -38,6 +38,23 @@ public static void main(String[] args) {
                     System.exit(-1);
                 } 
         }
+		// Now edit a Vec6 Property (inertia)
+		AbstractProperty inertiaProp = bdy.getPropertyByName("inertia");
+		Vec6 oldInertia = new Vec6();
+		for(int i=0; i<6; i++){
+			oldInertia.set(i, PropertyHelper.getValueVec6(inertiaProp, i));
+			System.out.println(oldInertia.get(i));
+		}
+		for(int i=0; i<6; i++){
+			PropertyHelper.setValueVec6(i, inertiaProp, i); // Bad inertia matrix but we're just testing set/get'
```

Comment:
Consider limiting the line length to 80 characters. However, I also think it's fine that we are more lenient about this the farther the code is from the C++ source files.